frontend/src/components/ComponentsTree/presenter.jsx

```
@@ -0,0 +1,111 @@
+import React from 'react'
```

ESLint

[error] 'React' is defined but never used.

(no-unused-vars)